### DISCUSSION TITLE
电报设置了代理依然登不上去，另外这个每天流量限制多少？

### DISCUSSION THREAD

**Discussion by fennafamearthlight** [SeaHOH/GotoX#209] [问答/咨询] [answered]
电报里设置了127.0.0.1：8087
依然登录不上去，网页访问全部正常。

另外这个每天限制流量吗？

**Comment by SeaHOH** [comment, marked as answer]
CFWorkers 不支持代理许多有严格安全设置的 APP，如电报。
每个 CF 账户都可使用 Workers 免费计划，每天十万个请求。
更多限制请查看 https://developers.cloudflare.com/workers/platform/limits
